sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -395,8 +403,8 @@ def tune(
                             client.V1Container(
                                 name=constants.DEFAULT_PRIMARY_CONTAINER_NAME,
                                 image=base_image,
-                                command=["bash", "-c"],
-                                args=[exec_script],
+                                command=["bash", "-c"] if callable(objective) else None,
+                                args=[exec_script] if callable(objective) else None,
```

Comment:
Also I'm not sure if we can assign `None` to `command` and `args` here when we use Docker image as objective.

As @andreyvelich shows an [example](https://github.com/kubeflow/katib/blob/master/examples/v1beta1/hp-tuning/random.yaml#L31-L36) for us, we sometimes need to pass `command` and `args` to the training container to execute python scripts with some parameters. 

Could you explain your idea in details so that I can understand more? WDYT👀 @akhilsaivenkata @andreyvelich

Reply:
I think, initially we can just allow user to set `image` as `objective` without `command` and `args`.
Similar to how we allow create training job using `base_image` parameter: https://github.com/kubeflow/training-operator/blob/master/sdk/python/kubeflow/training/api/training_client.py#L327C35-L327C45.